ipatests/test_xmlrpc/test_automember_plugin.py

```diff
@@ -851,6 +851,251 @@ def test_create_inclusive_with_group_badregextype(
+        automember_grp = request.getfixturevalue(automember_grp)
+        grp.ensure_exists()
+        automember_grp.ensure_exists()
+        automember_grp.add_condition(key=u'manager',
+                                     type=automember_grp.membertype,
+                                     inclusiveregex=[group_include_regex])
+        command = automember_grp.make_remove_condition_command(
+            key='badkey', type=automember_grp.membertype,
+            automemberinclusiveregex=[group_include_regex])
+        result = command()
+
+        expected = dict(
+            value=automember_grp.cn,
+            summary=u'Removed condition(s) from "%s"' % automember_grp.cn,
+            completed=0,
+            failed=dict(
+                failed=dict(
+                    automemberexclusiveregex=tuple(),
+                    automemberinclusiveregex=(u'badkey=%s' %
+                                              group_include_regex,),
+                )
+            ),
+            result=dict(
+                automemberinclusiveregex=[u'manager=%s' %
+                                          group_include_regex],
+            ),
+        )
+        assert_deepequal(expected, result)
+
+    @pytest.mark.parametrize(
+        "automember_grp, grp", [
+            ("automember_group", "group1"),
+            ("automember_hostgroup", "hostgroup1")
+        ])
+    def test_remove_inclusive_with_group_badregex(
+            self, automember_grp, grp, request):
+        grp = request.getfixturevalue(grp)
+        automember_grp = request.getfixturevalue(automember_grp)
+        grp.ensure_exists()
+        automember_grp.ensure_exists()
+        command = automember_grp.make_remove_condition_command(
+            key='manager', type=automember_grp.membertype,
+            automemberinclusiveregex=[u'badmscott'])
+        result = command()
+
+        expected = dict(
+            value=automember_grp.cn,
+            summary=u'Removed condition(s) from "%s"' % automember_grp.cn,
+            completed=0,
+            failed=dict(
+                failed=dict(
+                    automemberexclusiveregex=tuple(),
+                    automemberinclusiveregex=(u'manager=badmscott',),
+                )
+            ),
+            result=dict(
+                automemberinclusiveregex=[u'manager=%s' %
+                                          group_include_regex],
+            ),
+        )
+        assert_deepequal(expected, result)
+
+    @pytest.mark.parametrize(
+        "automember_grp, grp", [
+            ("automember_group", "group1"),
+            ("automember_hostgroup", "hostgroup1")
+        ])
+    def test_positive_remove_inclusive(
+            self, automember_grp, grp, request):
+        grp = request.getfixturevalue(grp)
+        automember_grp = request.getfixturevalue(automember_grp)
+        grp.ensure_exists()
+        automember_grp.ensure_exists()
+        automember_grp.make_remove_condition_command(
```

Comment:
The code *prepares* the command but does not execute it. The code should be instead:
```
        command = automember_grp.make_remove_condition_command(
            key='manager', type=automember_grp.membertype,
            automemberinclusiveregex=[u'mjohn'])
        command()
```